### DISCUSSION TITLE
Forum link on website is incorrect

### DISCUSSION THREAD

**Discussion by ghost** [stacked-git/stgit#427] [General]
It seems the forum links to a shady google group.

**Comment by kmerfeld** [comment]
I think the link is correct, it just got spammed. @jpgrayson it might be a good idea to archive the "forum"

**Reply by jpgrayson** [reply]
Thanks for the heads-up about this.

@ctmarinas owns that group and will need to take action.

I can remove the link from the StGit home page.
